hpx/parallel/algorithms/exclusive_scan.hpp

```diff
@@ -146,16 +150,16 @@ namespace hpx { namespace parallel { inline namespace v1
                     std::forward<ExPolicy>(policy),
                     make_zip_iterator(first, dest), count, init,
                     // step 1 performs first part of scan algorithm
-                    [op](zip_iterator part_begin, std::size_t part_size) -> T
+                    [op, conv](zip_iterator part_begin, std::size_t part_size) -> T
                     {
-                        T part_init = get<0>(*part_begin++);
+                        T part_init = hpx::util::invoke(conv, get<0>(*part_begin++));
```

Comment:
This additionally requires a check whether `part_begin` has not reached the end after the increment.